src/API/instance.ts

```diff
@@ -0,0 +1,8 @@
+import axios from "axios"
+
+export const instance = axios.create({
+    baseURL: "https://api.github.com/search/users/",
```

Comment:
baseURL should contain a basic domain without a pathname e.g. `https://api.github.com`. All other details like `/search/users` should be specified separately with every endpoint call.